### DISCUSSION TITLE
マスターキーの保管

### DISCUSSION THREAD

**Discussion by waonme** [concrnt/concrnt-square#9] [Q&A] [answered]
皆さんに質問です！ マスターキーはどう保管していますか？ 自分はメールの下書きに保管している他に、1Passwordに保管しています！

**Comment by ouroboros723** [comment, marked as answer]
現状、一番自分が情報を参照する場所（サービス）に放り投げてます。
（クラウドメモアプリとか）

でも流出が怖いのでできれば自前のサーバで管理したいですね。

**Reply by waonme** [reply]
よく使うサービスのセキュリティが優れていても、よく使うのでさまざまな端末からいつでもアクセス可能なのは諸刃の剣ですよね……！ 特別な認証が追加で個別にかけられる仕組みがあるようなサービスであれば適してそうだなと思いました。

自前のサーバーだと自分の場合はデータの維持とセキュリティの両面でなんとなくハードルが高めに感じますが勉強になりそうですね👍

**Comment by tigerwall** [comment]
ローカルのストレージとNotionの奥深くに保存することにしました
二つ用意しておけばどちらかがダメになってもどうにかなると思っていますがセキュリティ的にはNotionはほんのり危ないかなとも思っています(フリープランなので)

**Reply by waonme** [reply]
Notionには詳しくないのですが、有料プランだとセキュリティ面でより適切な機能が利用できるのでしょうか？

**Reply by tigerwall** [reply]
自分も若干あやふやだったので改めて確認したのですがビジネス以上だと（おそらく）より適切な機能が使えそう、といった温度感ですね・・・
フリーでも二段階認証があるので、セキュリティ的には十分そうです！
